frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -60,11 +60,21 @@ export default function AppNavbar({
                       Users
                     </NavDropdown.Item>
                   </NavDropdown>
-                  <Nav.Link as={Link} to="/moderate">
-                    Moderate
-                  </Nav.Link>
                 </>
               )}
+
+              {
+                // Stryker disable all: prevent mutation of authorization check
```

Comment:
You may not use a stryker exception for this.  You need to actually write tests for it.